spec/display-buffer-spec.coffee

```diff
@@ -147,19 +144,13 @@ describe "DisplayBuffer", ->
         it "includes hanging indent when breaking soft-wrap indentation into tokens", ->
           tokenizedLine = displayBuffer.tokenizedLineForScreenRow(4)
 
-          expect(tokenizedLine.tokens[0].value).toBe("  ")
+          expect(tokenizedLine.tokens[0].value).toBe("    ")
           expect(tokenizedLine.tokens[0].isSoftWrapIndentation).toBeTruthy()
 
-          expect(tokenizedLine.tokens[1].value).toBe("  ")
+          expect(tokenizedLine.tokens[1].value).toBe("   ") # hanging indent
           expect(tokenizedLine.tokens[1].isSoftWrapIndentation).toBeTruthy()
 
-          expect(tokenizedLine.tokens[2].value).toBe("  ") # hanging indent
```

Comment:
Yeah, that's a much better idea. I had changed the global tab length before because only one other test was using it, but I'll do it this way instead now that there are a lot of tests using 2-space tabs.